data/custom-formats.yaml

```diff
@@ -50,4 +50,7 @@ mx-mxc-uri:
 uri:
   title: URI
   url: http://tools.ietf.org/html/rfc3986
-  # no regex
```

Comment:
Did you mean to remove the `# no regex` comment, and also not add one for `email`.

And if so, why?

Reply:
Yes, because we updated the docs in #1977 to say that only custom formats need a regex. And the comment was removed there as well.